docs/articles_en/about-openvino/release-notes-openvino.rst

```diff
@@ -102,7 +102,6 @@ Support Change and Deprecation Notices
     We encourage developers to use the Neural Processing Unit (NPU) for low-powered systems
     like Intel® CoreTM Ultra or 14th generation and beyond.
   * OpenVINO C++/C/Python 1.0 APIs are deprecated and will be discontinued in the 2024.0 release.
-    Please use API 2.0 in your applications going forward to avoid disruption.
```

Comment:
I suppose we need to keep release notes for old releases, where both APIs are still there

Reply:
Have reverted it in the https://github.com/openvinotoolkit/openvino/pull/22884/commits/2479c43278578905cfb2f18f6f5d696852054442 commit.